packages/swapper/src/swappers/thorchain/ThorchainSwapper.ts

```diff
@@ -48,14 +58,19 @@ export class ThorchainSwapper implements Swapper<ChainId> {
         `${this.deps.midgardUrl}/pools`
       )
 
-      const supportedAssetIds = responseData.reduce<AssetId[]>((acc, midgardPool) => {
+      this.supportedSellAssetIds = responseData.reduce<AssetId[]>((acc, midgardPool) => {
         const assetId = adapters.poolAssetIdToAssetId(midgardPool.asset)
-        if (!assetId || !this.swapSupportedChainIds[fromAssetId(assetId).chainId]) return acc
+        if (!assetId || !this.sellSupportedChainIds[fromAssetId(assetId).chainId]) return acc
```

Comment:
Should we keep a consistent terminology? e.g `supportedSellChainIds` for the second

Reply:
its only used in this file and not passed around anywhere. I dont mind this name

Reply:
will just say ocd gang agrees with @gomesalexandre , but not blocking